index.html

```diff
@@ -0,0 +1,264 @@
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>Exhibitions</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-header">
+              <div class="card-header-img">
+                <img
+                  src="./assets/Images/forum-icon-black.PNG"
+                  alt="forum-icon"
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>Forum</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-header">
+              <div class="card-header-img">
+                <img
+                  src="./assets/Images/workshop-icon-black.PNG"
+                  alt="workshop-icon"
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>Workshop</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-header">
+              <div class="card-header-img">
+                <img
+                  src="./assets/Images/ignite-icon-black.PNG"
+                  alt="ignite-icon"
+                />
+              </div>
+            </div>
+            <div class="card-body">
+              <h3>CF Ignite</h3>
+              <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+            </div>
+          </div>
+        </div>
+        <button class="btn-conference">Join Conference Fora 2022</button>
+        <div class="card-bottom-txt">
+          <h3>see the whole program</h3>
+          <div class="underline"></div>
+        </div>
+      </section>
+      <section class="featured-speakers">
+        <div class="title">
+          <h2>Featured Speakers</h2>
+          <div class="underline"></div>
+        </div>
+        <div class="section-center">
+          <article class="featured-speakers-item">
+            <div class="speaker">
+              <img
+                src="./assets/Images/D.-SARAVANANA.jpeg"
+                alt="menu item"
+                class="photo"
+              />
+            </div>
+            <div class="item-info">
+              <header>
+                <h4>title here</h4>
+              </header>
+              <div class="item-detail">
+                <p class="item-text">
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  Repudiandae, sint quam. Et reprehenderit fugiat nesciunt
+                  inventore laboriosam excepturi! Quo, officia.
+                </p>
+                <p class="speaker-info"></p>
+              </div>
+            </div>
+          </article>
+        </div>
```

Comment:
- [ ] Your 'Featured Speakers' section does not have the design of the desktop template on screens between 768px and 1200px and it looks like the mobile version. Kindly make this section with 2 columns of 3 speakers on each like you did for screens larger than 1200px wide so it can match with the template.